test/functional/feature_addrman.py

```diff
@@ -44,7 +44,7 @@ def serialize_addrman(
 
 
 def write_addrman(peers_dat, **kwargs):
-    with open(peers_dat, "wb") as f:
+    with peers_dat.open(mode="wb") as f:
```

Comment:
ya i think in python3 both pathlib and os open use io.open so no difference, in general my thought was to remove `import os` from files so that there's an extra step of adding the import is involved in using `os.path.join` in the code just as a deterrent but ya I will revert this one.